pages/my-account.vue

```diff
@@ -1,72 +1,107 @@
 <template>
   <div id="my-account">
-    <SfBreadcrumbs class="breadcrumbs desktop-only" :breadcrumbs="breadcrumbs" />
-    <SfContentPages
-      title="My Account"
-      :active="activePage"
-      class="my-account"
-      @click:change="changeActivePage"
-    >
-      <SfContentCategory title="Personal Details">
-        <SfContentPage title="My profile" data-testid="my-profile">
-          <SfMyProfile
-            :account="account"
-            data-testid="my-profile-tabs"
-            @update:personal="account = { ...account, ...$event }"
-            @update:password="account = { ...account, ...$event }"
-          />
-        </SfContentPage>
-        <SfContentPage title="Shipping details" data-testid="shipping-details">
-          <SfShippingDetails
-            :account="account"
-            :countries="countries"
-            data-testid="shipping-details-tabs"
-            @update:shipping="account = { ...account, ...$event }"
-          />
-        </SfContentPage>
-        <SfContentPage title="Newsletter">
-          <SfMyNewsletter />
-        </SfContentPage>
-      </SfContentCategory>
-      <SfContentCategory title="Order details">
-        <SfContentPage title="Order history">
-          <SfOrderHistory :orders="account.orders" />
-        </SfContentPage>
-      </SfContentCategory>
-      <SfContentPage title="Log out" />
-    </SfContentPages>
+    <div>
+      <div>
+        <SfBar :title="barTitle" :back="true" class="title-bar" @click:back="goBack" />
+      </div>
+      <div v-show="activePage === 'Back'" class="profile-img-container">
+        <img class="profile-img-container__pic" :src="logo" alt="avatar" />
+      </div>
+      <div></div>
+    </div>
+    <div v-show="activePage === 'Back'">
+      <div class="header-text-font">My Account</div>
+      <SfAccordion
+        open=""
+        :multiple="false"
+        transition=""
+        show-chevron
+        @click:open="changeActivePage"
+      >
+        <SfAccordionItem class="kibo-sf-accordion-item" header="My profile">
+          <SfList>
+            <SfListItem>
+              <KiboMyProfile :user="account" />
+            </SfListItem>
+          </SfList>
+        </SfAccordionItem>
+        <SfAccordionItem class="kibo-sf-accordion-item" header="Shipping Address">
+          <SfList>
+            <SfListItem>
+              <UserSavedAddresses />
+            </SfListItem>
+          </SfList>
+        </SfAccordionItem>
+        <SfAccordionItem class="kibo-sf-accordion-item" header="Payment Method">
```

Comment:
I think we should use localization for "My Profile", "Shipping Address" and "Payment Method"